src/packages/admin-ui-components/src/alert/styles.module.css

```diff
@@ -0,0 +1,45 @@
+.container {
+    display: flex;
+    flex-direction: row;
+    padding: 6px 16px;
+}
+
+svg {
+    width: 24px;
+    height: 24px;
+}
+
+.error {
+    background-color: #ce5e5e;
+}
+
+.error svg {
+    fill: red;
+}
+
+.success {
+    background-color: #abe9b8;
+    color: green;
+}
+
+.success svg {
+    fill: green;
+}
+
+.info {
+    background-color: #abbce9;
```

Comment:
I wonder if we should start defining these colours as vars. What do you think?

Reply:
Done.